### PR TITLE
[Improve][Doc] Improve the connector-v2 develop doc

### PR DESCRIPTION
**Pull request by YOMO-Lee**
Supplementary explanations have been provided for the methods and classes that must be implemented during the source development process

### CODE REVIEW

File: seatunnel-connectors-v2/README.md [hunk 1]
```diff
@@ -49,14 +49,19 @@ own connectors, you need to follow the steps below.
...
 
 4.add connector info to plugin-mapping.properties file in seatunnel root path.
 
 5.add connector dependency to seatunnel-dist/pom.xml, so the connector jar can be find in binary package.
 
+6.There are several classes that must be implemented on the source side, namely {connector name} Source, {connector name} SourceFactory, {connector name} SourceReader. Please refer to other connectors for details
+
+7.{Connector Name} SourceFactory needs to be annotated with the **@AutoService (Factory.class)** annotation on the class name, and in addition to the required methods, an additional 'creatSource' method needs to be rewritten
+
+8.{Connector Name} Source needs to override the getProducedCatalogTables method
```
Comment (Hisoka-X) [inline seatunnel-connectors-v2/README.md]: Please add sink too

**Review by Hisoka-X — COMMENTED** [review]
Thanks @YOMO-Lee . This PR looks great.

**Review by liunaijie — DISMISSED** [review]

File: seatunnel-connectors-v2/README.md (hunk 1 above)
Comment (YOMO-Lee) [reply seatunnel-connectors-v2/README.md]: But I have only done source development and not sink development, so I am not sure if what I am writing is correct

**Review by hailin0 — DISMISSED** [review]

**Review by Hisoka-X — APPROVED** [review]

**Comment by YOMO-Lee** [comment]
@hailin0 @liunaijie please review again ,thx

**Review by dailai — APPROVED** [review]

**Pull request merged**